### DISCUSSION TITLE
VulkanSceneGraph-1.1.8, vsgXchange-1.1.4, vsgQt-0.40, vsgImGui-0.6.0 and vsgExamples-1.1.6 releases tagged

### DISCUSSION THREAD

**Discussion by robertosfield** [vsg-dev/VulkanSceneGraph#1337] [General]
After a bit delay I'm finally back on the release wagon and have tagged developer releases for all the main VulkanSceneGraph projects.  The releases are:

* [VulkanSceneGraph-1.1.8](https://github.com/vsg-dev/VulkanSceneGraph/releases/tag/v1.1.8)
* [vsgXchange-1.1.4](https://github.com/vsg-dev/vsgXchange/releases/tag/v1.1.4)
* [vsgQt-0.4.0](https://github.com/vsg-dev/vsgQt/releases/tag/v0.4.0)
* [vsgImGui-0.6.0](https://github.com/vsg-dev/vsgImGui/releases/tag/v0.6.0)
* [vsgExamples-1.1.6](https://github.com/vsg-dev/vsgExamples/releases/tag/v1.1.6)

The biggest changes/additions are the addition of[ vsg::PolytopeIntersector](https://github.com/vsg-dev/VulkanSceneGraph/blob/master/include/vsg/utils/PolytopeIntersector.h) and a refactor of how the compile traversal manages data transfer, shifting the task to dedicated [vsg::TransferTask](https://github.com/vsg-dev/VulkanSceneGraph/blob/master/include/vsg/app/TransferTask.h) class that is faster and lowers latency of database paging.

vsgImGui has been updated to work with the latest ImGui-1.91.4 release, and there have been various refinements to vsgXchange, vsgQt and vsgExamples.

**Comment by gwaldron** [comment]
vsg 1.1.8 PR for vcpkg: https://github.com/microsoft/vcpkg/pull/42621
